docs/install.rst

```diff
@@ -326,6 +320,35 @@ Important to note is that packages on `conda` and `pip` are typically *not* comp
 Frequently asked questions
 --------------------------
 
+**Why are the plots in my notebook not interactive?**
+
+Jupyter notebooks support the use of interactive figures.
+To enable interactive plots, you have to invoke the correct `magic commands <https://ipython.readthedocs.io/en/stable/interactive/magics.html>`_ in the notebook.
+When using Jupyter notebook, the following command will switch the `matplotlib` backend from the inline one (which renders images) to the interactive backend::
+
+    %matplotlib widget
+
+When using JupyterLab, `ipympl` is the only backend that provides interactive plots with Pylake.
+
+.. note::
+
+    Switching plotting backends typically requires you to restart the Jupyter kernel.
+
+**I tried using interactive plots using Jupyter notebook, but get the error `Javascript Error: IPython is not defined`**
+
```

Comment:
I would start with an instruction how to solve the issue, an then add the explanation. So, something like:

Starting from Pylake `1.4.0`, Pylake uses a more recent version of Jupyter Notebook (`notebook >=7)`. This newer version requires a different backend: `%matpltolib notebook` has to be replaced by `%matplotlib widget`.

Note that switching between interactive backend requires a kernel restart (select kernel -> restart from the menu at the top of the Jupyter Notebook)

The reason for the update is that the backend which handled interactive figures in `notebook<7` is no longer supported by Jupyter Notebooks. See also the matplotlib documentation ...

Reply:
Yeah, good idea to start with the solution.

I'd rephrase it a little bit, because Pylake didn't really switch anything internally though. You could in principle install with `pip` without `[notebook]`, install `notebook<7` yourself and get the old situation. It would still work.

All we are doing is updating what we install by default on `conda` (what we ship with essentially).

```
Starting from Pylake `1.4.0`, Pylake ships with a more recent version of Jupyter Notebook (`notebook >=7`).
This newer version requires a different backend when using interactive figures: `%matplotlib notebook` has to be replaced by `%matplotlib widget`.
Note that switching between interactive backends requires a kernel restart (select kernel -> restart from the menu at the top of the Jupyter Notebook).

The reason for this change is that the backend which handled the interactive figures in `notebook<7` is no longer supported by Jupyter notebooks.
Moving forward, `ipympl` will be used for interactive figures.
See also the `matplotlib documentation <https://matplotlib.org/stable/users/explain/figure/interactive.html#jupyter-notebooks-jupyterlab>`_ on this.
```